src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: 'delight';
+  src: url('./Delight.ttf') format('ttf'), url('./Delight.ttf') format('ttf');
+  font-weight: 600;
+  font-style: normal;
+}
+
+body {
+  background-color: #f6e8ea;
+}
+
+#container {
+  width: 100%;
+  max-width: 25%;
+  margin: auto;
+  background-color: white;
+  box-shadow: 9px 7px 13px -2px rgba(136, 130, 130, 1);
+  -webkit-box-shadow: 9px 7px 13px -2px rgba(136, 130, 130, 1);
+  -moz-box-shadow: 9px 7px 13px -2px rgba(136, 130, 130, 1);
+}
+
+ul {
+  list-style: none;
+  padding: 0%;
+  margin: auto;
+}
+
+ul li {
+  height: 3vw;
+  width: 100%;
+  text-align: left;
+  max-width: 100%;
+  align-items: center;
+  display: inline-flex;
+  border-bottom: 1px solid grey;
+}
+
+ul li button {
+  float: right;
+  margin: auto;
+  margin-right: 2%;
+  background: none;
+  border: none;
+  padding: 0%;
+}
+
+ul li p {
+  font-family: 'delight', 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  margin-left: 1vw;
+  padding: 0%;
+  font-size: 1.5vw;
+}
+
+ul li input {
+  font-family: 'delight', 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  margin-left: 1vw;
+  text-align: left;
+  font-size: 1.5vw;
+}
+
+#clear {
+  text-align: center;
+}
+
+#task,
+#task1 {
+  width: 100%;
+  max-width: 20vw;
+  border: none;
+  outline: none;
+}
+
+#task,
+#task1:focus {
```

Comment:
It would be better to use classes instead of ids for styling your tasks, since you have multiple tasks on your page, and ids should be unique on your page. 
![image](https://user-images.githubusercontent.com/51750930/135919271-e86fdd17-ecaa-4be5-91ed-cbd6f9362dbc.png)